frontend/src/components/blocks/Template/Template.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ReactElement } from 'react';
+import dynamic from 'next/dynamic';
+import { TemplateBlockContent, TemplateBlockProps } from '../../../blocks/TemplateBlock/TemplateBlock';
+
+const Button = dynamic(import('../Button/Button'));
+
+export type TemplateProps = {
+    blocksData: Omit<TemplateBlockProps, 'content'> & Omit<TemplateBlockContent, 'id' | '__typename'>;
+};
+
+const Template = ({ blocksData: { template } }: TemplateProps): ReactElement => {
+    if (!template || !template.data || !template.data.attributes || template.data.attributes.content.length === 0)
```

Comment:
@JakubVavra tu by to slo pomocou optional chaining operatoru zjednodusit tu podmienku. Zaroven prosim pri podmienkach pouzivajme vsade blokovy zapis, takto bez {} to je niekedy zradne a vedie to k chybam, radsej by som to nepouzival ak to nie je vylozene nutne.